src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,267 @@
+                result += "\t" + newEvent.toString() + "\n";
+                result += "Now you have " + TaskList.getCounter() +
+                        " task(s) in the list.\n";
+                return result;
+            }
+        }
+    }
+
+    /**
+     * Prints out the list of tasks saved by Jarvis
+     */
+    public static String parseList() {
+        int num = 1;
+        String result = "";
+        for (int i = 0; i < TaskList.getTaskList().size(); i++) {
+            result += num + "." + TaskList.getTaskList().get(i).toString() + "\n";
+            num++;
+        }
+        return result;
+    }
+
+    /**
+     * Prints out the list of tasks set for/due today and all todo tasks
+     */
+    public static String parseToday() {
+        String result = "";
+        result += "Tasks scheduled for today are: \n";
+        int num = 1;
+        for (int i = 0; i < TaskList.getTaskList().size(); i++) {
+            if (TaskList.getTaskList().get(i) instanceof Todo) {
+                result += "\t" + num + "." + TaskList.getTaskList().get(i).toString() + "\n";
+                num++;
+            } else {
+                int currYear = LocalDateTime.now().getYear();
+                int currMonth = LocalDateTime.now().getMonthValue();
+                int currDate = LocalDateTime.now().getDayOfMonth();
+                LocalDateTime start = LocalDateTime.of(currYear, currMonth, currDate, 0, 0);
+                LocalDateTime end = LocalDateTime.of(currYear, currMonth, currDate, 23, 59);
+
+                if (TaskList.getTaskList().get(i) instanceof Deadline &&
+                        (((Deadline) TaskList.getTaskList().get(i)).getDeadline()).isAfter(start) &&
+                        (((Deadline) TaskList.getTaskList().get(i)).getDeadline()).isBefore(end)) {
+                    result += "\t" + num + "." + TaskList.getTaskList().get(i).toString() + "\n";
+                    num++;
+                } else if (TaskList.getTaskList().get(i) instanceof Event &&
+                        (((Event) TaskList.getTaskList().get(i)).getEventStart()).isAfter(start) &&
+                        (((Event) TaskList.getTaskList().get(i)).getEventStart()).isBefore(end) &&
+                        (((Event) TaskList.getTaskList().get(i)).getEventEnd()).isAfter(start) &&
+                        (((Event) TaskList.getTaskList().get(i)).getEventEnd()).isBefore(end)) {
+                    result += "\t" + num + "." + TaskList.getTaskList().get(i).toString() + "\n";
+                    num++;
+                }
+            }
+        }
+        if (num == 1) {
+            result += "\tLooks like there is nothing due today!\n";
+        }
+        return result;
+    }
+
+    public static String parseFind(String search) {
+        String result = "";
+        result += "Here are the matching tasks in your list:\n";
+        int num = 1;
+        search = search.substring(5);
+        for (int i = 0; i < TaskList.getTaskList().size(); i++) {
```

Comment:
Would it perhaps be better to abstract getting size into a separate method so that other developers can't modify your task list improperly?